### PR TITLE
grpcproxy: fix double create event

### PR DESCRIPTION
**Pull request by xiang90**
/cc @heyitsanthony

### CODE REVIEW

File: proxy/grpcproxy/watcher_groups.go [hunk 1]
```diff
@@ -43,6 +43,14 @@ func (wgs *watchergroups) addWatcher(rid receiverID, w watcher) {
 		rev := wg.add(rid, w)
 		wgs.idToGroup[rid] = wg
 
+		// The group is newly created, the create event has not been delievered
```
Comment (heyitsanthony) [inline proxy/grpcproxy/watcher_groups.go]: s/delivered/delivered

File: proxy/grpcproxy/watcher_groups.go [hunk 2]
```diff
@@ -43,6 +43,14 @@ func (wgs *watchergroups) addWatcher(rid receiverID, w watcher) {
...
+		// to this group yet.
+		// We can rely on etcd server to deliever the create event.
```
Comment (heyitsanthony) [inline proxy/grpcproxy/watcher_groups.go]: s/deliever/deliver/

File: proxy/grpcproxy/watcher_groups.go [hunk 3]
```diff
@@ -43,6 +43,14 @@ func (wgs *watchergroups) addWatcher(rid receiverID, w watcher) {
...
+		// Or we mgiht end up sending created event twice.
```
Comment (heyitsanthony) [inline proxy/grpcproxy/watcher_groups.go]: s/mgiht/might

File: proxy/grpcproxy/watcher_groups.go [hunk 4]
```diff
@@ -43,6 +43,14 @@ func (wgs *watchergroups) addWatcher(rid receiverID, w watcher) {
...
+		if rev == 0 {
+			return
```
Comment (heyitsanthony) [inline proxy/grpcproxy/watcher_groups.go]: move inside here?

**Comment by xiang90** [comment]
@heyitsanthony PTAL.

**Comment by heyitsanthony** [comment]
lgtm

**Pull request merged**